research/schooltests/sql.txt

```diff
@@ -0,0 +1,126 @@
+Erstelle eine Übersicht mit allen Songtiteln und deren Erscheingungsdatum, sortiert nach dem Erscheinungsdatum (absteigend).
```

Comment:
What do you think about:

"Erstelle eine Übersicht beginnend mit dem zuletzt veröffentlichten Song, welche alle Songtitel und deren Erscheinungsdatum enthält"

would require more cognitive work cause the named requirements (Columns, order by) are not in the correct order in the task. And "Order" isn't name explicitly in the task text.

Reply:
Rly dont know.. cause this isnt our research base point. We investigate others features..

Reply:
okay you are right. But might be an interesting thing to research 🤔